cmd/envsubst/main.go

```diff
@@ -14,6 +14,7 @@ import (
 var (
 	input    = flag.String("i", "", "")
 	output   = flag.String("o", "", "")
+	noDigit  = flag.Bool("d", false, "")
```

Comment:
```suggestion
	noDigit  = flag.Bool("no-digit", false, "")
```
Let's stay consistent with the other restrictions (they start with `-no`).

Reply:
Agree, looks more consistent with -no-digit. I also updated the help message and the readme accordingly.